### ISSUE TITLE
Feature Request: Training and Validation (Loss vs Epoch) to Avoid Overfitting

### ISSUE THREAD

**Issue by UncleBob2** [Roffild/RoffildLibrary#6]
Roffild,

Great job in sharing your work with everyone. I am wondering if you are working on ways to avoid overfitting in ML for designing trading strategies. In the past, there were Walk Forward Optimizations. There must be a way to implement EA optimization so that we don't over curve fit our strategies.

**Comment by Roffild** [comment]
In [sklearn.metrics](https://scikit-learn.org/stable/modules/model_evaluation.html) are the best metrics to evaluate the model. It is enough to dump the test period and the verification period.

I use PythonDLL to check the final model.
